### PR TITLE
Feature/drop ofec sched b

### PR DESCRIPTION
**Pull request by fecjjeng**
## Summary (required)

- Resolves $3098 drop ofec_sched_b_xxxx tables and related process

ofec_sched_b_master and its children tables are dropped.  The queue table, database procedures/triggers and the scheduled tasksused to handle their updates are dropped too.  SQL statements in the Sample_db.sql that populated these tables are removed too.

## How to test the changes locally

- After Flyway migration completed, the ofec_sched_b related objects listed in the migration file V0087 should not exist in the local database anymore.

## Impacted areas of the application
List general components of the application that this PR will affect:

-  After successfully switching to use the fec_fitem_sched_b tables, these ofec_sched_b table are not in use anymore and will not have impact on API.

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------
fix/other_pr | [link]()
feature/other_pr | [link]()

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -195,7 +195,7 @@ def refresh_itemized():
     """These are run nightly to refresh the itemized schedule A and B data."""
 
     refresh_itemized_a()
-    refresh_itemized_b()
+    # refresh_itemized_b()
```
Comment (vrajmohan) [inline manage.py]: Why not just remove the line instead of commenting it out? Same comment applies to other places as well.

File: manage.py [hunk 2]
```diff
@@ -214,18 +214,18 @@ def refresh_itemized_a():
 
     logger.info('Finished updating Schedule A.')
 
-@manager.command
-def refresh_itemized_b():
-    """Used to refresh the itemized Schedule B data."""
-    logger.info('Updating Schedule B...')
-    message = partition.SchedBGroup.process_queues()
+# @manager.command
```
Comment (vrajmohan) [inline manage.py]: Remove entire method instead of commenting

**Comment by vrajmohan** [comment]
It would make sense to use `git rebase -i` to create a clean history. There are some repeated commit messages and messages like "update migration file name to V0084" and "rename migration file V0085 to V0087" which could be eliminated.

**Comment by fecjjeng** [comment]
manage.py had been updated to remove the lines that are not needed anymore.
The branch had been rebased to combine the commits to make it clean.

**Comment by vrajmohan** [comment]
Nitpicking, but the commit message could be cleaned up to be more concise.

File: data/migrations/V0091__drop_ofec_sched_b_objects.sql [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+    child_table_name TEXT;
+
+    schedule TEXT;
+
+BEGIN
+
+    FOR cycle IN first_cycle_end_year..last_cycle_end_year BY 2 LOOP
+
+        FOREACH schedule IN ARRAY ARRAY['a'] LOOP
```
Comment (vrajmohan) [inline data/migrations/V0091__drop_ofec_sched_b_objects.sql]: Does it make sense to remove the entire loop and use:
```
 master_table_name = 'ofec_sched_a_master';
```,
etc?

File: data/migrations/V0091__drop_ofec_sched_b_objects.sql [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+ALTER FUNCTION public.add_partition_cycles(numeric, numeric)
+  OWNER TO fec;
+
+
+DROP FUNCTION IF EXISTS public.finalize_itemized_schedule_b_tables(numeric, numeric, boolean, boolean);
+
+-- ------------------
+-- FUNCTION disclosure.finalize_itemized_schedule_b_tables
+-- ------------------
+CREATE OR REPLACE FUNCTION disclosure.finalize_itemized_schedule_b_tables(
```
Comment (vrajmohan) [inline data/migrations/V0091__drop_ofec_sched_b_objects.sql]: Should this function be dropped?

File: tests/test_integration.py [hunk 5]
```diff
@@ -484,9 +484,9 @@ def test_add_itemized_partition_cycle(self):
         manage.add_itemized_partition_cycle(3002, 2)
         expected_tables = {
             "ofec_sched_a_3001_3002",
-            "ofec_sched_b_3003_3004",
-            "ofec_sched_a_3001_3002",
-            "ofec_sched_b_3003_3004",
+            "ofec_sched_a_3003_3004",
+            # "ofec_sched_b_3001_3002",
+            # "ofec_sched_b_3003_3004",
```
Comment (vrajmohan) [inline tests/test_integration.py]: Can we remove these lines?

File: webservices/partition/__init__.py [hunk 6]
```diff
@@ -1,2 +1,2 @@
 from .sched_a import SchedAGroup  # noqa
-from .sched_b import SchedBGroup  # noqa
+# from .sched_b import SchedBGroup  # noqa
```
Comment (vrajmohan) [inline webservices/partition/__init__.py]: Can we remove this line?

File: data/migrations/V0091__drop_ofec_sched_b_objects.sql (hunk 3 above)
Comment (fecjjeng) [reply data/migrations/V0091__drop_ofec_sched_b_objects.sql]: The whole function public.add_partition_cycles will be dropped in the next issue that drops ofec_sched_a tables and process, which we plan to implement in next Sprint

File: data/migrations/V0091__drop_ofec_sched_b_objects.sql (hunk 4 above)
Comment (fecjjeng) [reply data/migrations/V0091__drop_ofec_sched_b_objects.sql]: This function had been rewritten to work on the new set of fec_fitem_sched_b tables.

**Comment by fecjjeng** [comment]
Files tests/test_integration.py and webservices/partition/__init__.py had been updated to remove the lines as recommended.
The commit messages had been cleaned up too.

**Pull request merged**